frontend/src/components/GoalForm/ResourceRepeater.js

```diff
@@ -19,10 +20,11 @@ export default function ResourceRepeater({
   status,
   isOnReport,
   isLoading,
+  goalStatus,
 }) {
   const resourcesWrapper = useRef();
 
-  const readOnly = status === 'Suspended' || (status === 'Not Started' && isOnReport);
+  const readOnly = status === 'Suspended' || (status === 'Not Started' && isOnReport) || (status === 'Completed' && goalStatus === 'Closed');
```

Comment:
Again do we care about 'In Progress' here?

Reply:
This logic refects the mockup